snippets/dashboard-config.mdx

```diff
@@ -7,11 +7,11 @@ Setting this value will change the port that Tyk Dashboard listens on. Default:
 ### tyk_api_config
 This section contains details for a Tyk Gateway node that the Tyk Dashboard can speak to. The Dashboard controls Tyk using the Gateway API and only requires visibility to one node, so long as all nodes are using the same API Definitions.
 
+{{< note success >}}
+**Note**
 
-<Note>
 If the Dashboard cannot see a Tyk node, key management functions will not work properly.
-</Note>
-
+{{< /note >}}
```

Comment:
`{{< note success >}}` is the old format for writing a Note in the Hugo platform. As we have migrated to Mintlify, we need to use the <Note> block.

Can you perform a find-and-replace operation in the codebase?